helm/eirini/templates/bits.yaml

```diff
@@ -10,10 +10,17 @@ data:
       level: debug
     private_endpoint: "https://bits.{{ .Release.Namespace }}.svc.cluster.local"
     {{- if .Values.services.loadbalanced }}
-    public_endpoint: https://registry.{{ index .Values.env.DOMAIN }}:6666
```

Comment:
If you want to remove port here, you have to modify service definition
https://github.com/cloudfoundry-incubator/eirini-release/pull/42/files#diff-7caec9ea445e7a764de57dbf83c95c22L117

Reply:
@alex-slynko I've removed it and force-pushed. Note that the port in this line is not really used to specify on which port the Bits-Service is listening. But to keep the change small and avoid further confusion I've removed it.